repos/system_upgrade/el8toel9/actors/checkblacklistca/tests/component_test_checkblacklistca.py

```diff
@@ -0,0 +1,146 @@
+from leapp import reporting
+from leapp.models import BlackListCA, BlackListError, Report
+
+
+def test_actor_execution_empty(current_actor_context):
+    current_actor_context.feed()
+    current_actor_context.run()
+    assert not current_actor_context.consume(Report)
+
+
+def test_actor_error_entry(current_actor_context):
+    current_actor_context.feed(
+        BlackListError(
+            sourceDir="/blacklist",
+            targetDir="/blocklist",
+            error="Can't list /blacklist"
+        )
+    )
+    current_actor_context.run()
+    r = current_actor_context.consume(Report)
+    assert r
+    assert 'Could not access blacklist directory' in r[0].report['title']
+    assert reporting.Groups.INHIBITOR in r[0].report['groups']
```

Comment:
You can use `is_inhibitor` helper from `from leapp.utils.report import is_inhibitor`

`assert is_inihibitor(r[0].report)`

Reply:
ok, thanks